README.md

```diff
@@ -2,4 +2,100 @@
+    - Run in browsers: [fsrs-browser](https://github.com/open-spaced-repetition/fsrs-browser)
+- TypeScript
+  - Scheduler (v5): [ts-fsrs](https://github.com/open-spaced-repetition/ts-fsrs)
+- Go
+  - Scheduler (v5): [go-fsrs](https://github.com/open-spaced-repetition/go-fsrs)
+- Dart
+  - Scheduler (v4.5): [dart-fsrs](https://github.com/open-spaced-repetition/dart-fsrs)
+- Swift
+  - Scheduler (v4.5): [swift-fsrs](https://github.com/open-spaced-repetition/swift-fsrs)
+- Clojure/ClojureScript
+  - Scheduler (v4): [cljc-fsrs](https://github.com/open-spaced-repetition/cljc-fsrs)
+- Ruby
+  - Scheduler (v4): [rb-fsrs](https://github.com/open-spaced-repetition/rb-fsrs)
+- Kotlin
+  - Scheduler (v4): [android-fsrs](https://github.com/open-spaced-repetition/android-fsrs)
+- Elixir
+  - Scheduler (v4): [ex_fsrs](https://github.com/open-spaced-repetition/ex_fsrs)
+
+## Application
+
+### Flashcard
+
+- #### Anki (https://apps.ankiweb.net/)
+
+  Free and open source content-agnostic flashcard application for Windows, Mac, Linux, iPhone, and Android. Supports images, audio, videos, and scientific markup (via LaTex). Offers free synchronization service using AnkiWeb, with community shared add-ons and decks.
+
+  - FSRS available as an opt-in feature replacing the default SM-2 algorithm.
+  - In addition to feature metioned above, this [add-on](https://ankiweb.net/shared/info/759844606) helps with reshedule, postpone, advance, and balance cards, as well as enabling Easy Days scheduling, among other features.
+
+- #### MaiMemo (https://www.maimemo.com/)
+
+  Chinese mobile flashcard application for studying English.
+
+### Note-taking
+
+- #### RemNote (https://www.remnote.com/)
+
+  Multiplatform note-taking application with flashcard creation in single click. Has an active community with student-made exam materials on various topics. Available for offline and online usage.
+
+  RemNote integrated FSRS v4.5 to its scheduling system starting from [RemNote 1.16](https://feedback.remnote.com/changelog/remnote-1-16-ultimate-spaced-repetition).
+
+- #### Obsidian (https://obsidian.md/)
+
+  A personal notes, journaling, knowledge base, and project managament application with graph view and canvas diagram note. Connect your notes via hyperlinks and install many community plugins made for Obsidian.
+
+  FSRS available as extension for Obsidian: [obsidian-spaced-repetition-recall](https://github.com/open-spaced-repetition/obsidian-spaced-repetition-recall).
+
+### Others
+
+- #### Chessbook (https://chessbook.com/)
+
+  Mobile and web chess repertoire study application, combining multiple sources from YouTube, Lichess, ChessBase, and books to create a custom personal chess repertoire.
+
+  Chessbook overhauled its spaced repetition system using FSRS v4.5 and saw [20-30% cut down on reviews count](https://x.com/chessbookcom/status/1805137108991946775).
+
+## Related
+
+### Algorithm
```

Comment:
I wanted to outline the differences between these algorithms and FSRS. I adopted this from Expertium's Abridged History of Spaced Repetition.
@Expertium perhaps you can help me fill some information here?

Reply:
1) I don't recommend calling FSRS-5 v5. LMSherlock specifically wanted to chaneg the naming convention from "FSRS vX" to "FSRS-X".
2) MaiMemo doesn't use FSRS. Their algorithm is similar, but not the same.
3) "Chessbook overhauled its spaced repetition system using FSRS v4.5 and saw 20-30% cut down on reviews count" isn't true. They quote my figure, which is turn is just a rough estimation based on simulations. We still don't have real data on how much FSRS reduces the workload.
4) You skipped a lot of SM algorithms: SM-6, SM-8, SM-11 and SM-15. You can find their descriptions on supermemo.guru.
5)

Reply:
For the naming convention, can we call it "v5" like what I already did in "Implementation" section? I think it's better to have a shorthand to avoid repeating the name over and over when it's unnecessary.

So that would be "FSRS-5" and "v5" respectively, without "FSRS v5".

Reply:
My guess is that LMSherlock would disapprove of that. But idk, ask him.

Reply:
> You skipped a lot of SM algorithms: SM-6, SM-8, SM-11, SM-15 and SM-18. You can find their descriptions on supermemo.guru.

Are there any important improvements for them to be mentioned? I do aware of their existence, but I don't think listing everything is a good idea.

Reply:
SM-18 simplified the calculation of difficulty, and that's it. As for the other ones, it's fine if you don't mention them, I don't think this is super critical.

Reply:
Please add two papers related to FSRS.

https://dl.acm.org/doi/10.1145/3534678.3539081

https://ieeexplore.ieee.org/document/10059206/